web/themes/custom/digital_gov/templates/node/node--topics.html.twig

```diff
@@ -179,11 +178,13 @@
     {# Display remaining resources, events and news for topic collection #}
     {% if is_first_page %}
       <section class="grid-container-desktop usa-section">
-        [placeholder remaining resources]
-        {# Only show these sections when on page 1, otherwise hide to show only More News and Events #}
-        {# $resource_settings := ( dict "header_title" (printf "Resources on %s" .Title) "header_size" 2 "variant" "media" "list_header" 3) #}
-        {# NOT-MIGRATED #}
-        {# partial "partials/core/collection/collection.html" (dict "collection_data" $resources "settings" $resource_settings ) #}
+        {# Shows all resources tagged to this topic, may repeat resources shown under featured resource. #}
+        {% if drupal_view_result('resources_for_topic', 'default', node.nid.value) is not empty %}
+          <div class="dg-collection">
+            <h2 class="dg-collection__header">Resources on {{ node.label }}</h2>
+            {{ drupal_view('resources_for_topic', 'default', node.nid.value) }}
+          </div>
+        {% endif %}
 
         {# $services_settings := ( dict "header_title" "Tools and Services" "header_size" 2 "variant" "media" "list_header" 3) #}
         {# NOT-MIGRATED #}
```

Comment:
Let's remove this commented out code because the services component is deprecated and will not be migrated.